src/conduit/controllers/article.cljs

```diff
@@ -22,3 +22,29 @@
 
 (defmethod control :update [_ [id transform data] state]
   {:state (merge state (transform data))})
+
+(defmethod control :save [_ [{:keys [title description body tagList]} token id] state]
+  (let [http-params (if id
```

Comment:
it would be better to make conditional assignments inside of the `:http` map directly, without creating intermediate values and merging them together